setup.py

```diff
@@ -119,9 +116,13 @@
 # For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
 if 'darwin' in sys.platform and PY3:
   mac_target = sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')
+  # The working computer's mac os x version defines what frameworks are
+  # available to build against.
+  mac_version = platform.mac_ver()[0]
```

Comment:
Can a mac not build against older frameworks?  I'm concerned about us compiling binaries against a high ```MACOSX_DEPLOYMENT_TARGET```.  I'm pretty sure our macos jenkins workers are post 10.7, but they are able to build against that target.